xtable-core/src/test/java/org/apache/xtable/delta/TestDeltaValueConverter.java

```diff
@@ -82,6 +82,15 @@ void parseWrongDateTime() throws ParseException {
     assertThrows(ParseException.class, () -> strictDateFormat.parse(wrongDateTime));
   }
 
+  @ParameterizedTest
+  @MethodSource("nonNumericValuesForColStats")
+  public void formattedDifferentNonNumericValuesFromDeltaColumnStat(
+      Object fieldValue, InternalSchema fieldSchema, Object expectedDeltaValue) {
+    Object internalRepresentation =
+        DeltaValueConverter.convertFromDeltaColumnStatValue(fieldValue, fieldSchema);
+    assertEquals(internalRepresentation, expectedDeltaValue);
```

Comment:
Do we also need to test transforming the internal representation into the delta stats?

Reply:
Good question, we suppose the internal representation always has correct field value which corresponding to field schema, and existing tests have already covered the case.